docker-compose.yml

```diff
@@ -55,7 +55,7 @@ services:
       MW_WG___UPLOADS:
       <<: *wikibase_extra_variables
     labels:
-      - traefik.http.routers.service-wikibase.rule=Host(`portal.mardi4nfdi.de`)
+      - traefik.http.routers.service-wikibase.rule=Host(`portal.mardi4nfdi.de`,`swmath.portal.mardi4nfdi.de`)
```

Comment:
Is the intention here to route the existing wikibase container to an additional swmath.portal.mardi4nfdi url ?

Reply:
Yes.